supportedBackends/tes/src/test/scala/cromwell/backend/impl/tes/TesTaskSpec.scala

```diff
@@ -47,4 +57,35 @@ class TesTaskSpec extends AnyFlatSpec with CromwellTimeoutSpec with Matchers wit
         Resources(None, None, None, Option(false), None, Option(Map.empty[String, Option[String]])
     )
   }
+
+
+  it should "copy labels to tags" in {
```

Comment:
Still an awful lot of setup, but easier than building a suite for the TES job execution actor just to test this function.